### PR TITLE
Dummy Experiment

### PR DESCRIPTION
**Pull request by Claralein**
Server neue socket.on Befehle hinzufegügt & neuen Test-Odner anglegt mit .js für Befehlsüberprüfung

### CODE REVIEW

File: test/dummy_exp.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    console.log(socket.id); // Outputs undefined
+ */
+});
+
+socket.on("message", () => {
+    console.log("Hello World!");
+    socket.emit('message', ("beetlebum", "Hello World!"));       //Output-> beetlebum: Hello World!
+});
+
+socket.on("rotationControl", () => {
```
Comment (jkbeetle) [inline test/dummy_exp.js]: please use events as defines in the wiki: https://github.com/Lichtwerkstatt/XRTL_SPA/wiki/Communication-Protocol
for example for rotation 
event "command"
payload: {userId, componentId, controlId, command {steps}}

File: test/dummy_exp.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    console.log("user1234", "km100_1", "top", 100);
+    socket.emit('rotationControl', { username: "user1234", componentId: "km100_1", controlId: "top", steps: 100 });
```
Comment (jkbeetle) [inline test/dummy_exp.js]: on rotation:
-  emit status update with busy true
- wait a certain amount of time to simulate duration of rotation
- emit status update with busy false and updated rotation

File: test/dummy_exp.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+});
+
+socket.on("rotationStatus", () => {
+    console.log("km100_1", true, 100, 180, 200);
+    socket.emit('rotationStatus', { componentId: "km100_1", busy: true, top: 180, bottom: 200 });
+
+});
+
+socket.on("connect", () => { });
+
```
Comment (jkbeetle) [inline test/dummy_exp.js:71]: we need global rotation status of dummy components in the dummy experiment.
For now: two KM100 (ids: KM100_1 and KM100_2) with each to stepper motors ("top" and "bootom")

**Review by jkbeetle — CHANGES_REQUESTED** [review]
:) testing the review functionality of Github

**Comment by Claralein** [comment]
Änerungen vorgenommen.
Bitte Überprüfen, ob sie den Forderungen gerecht werden :)

**Pull request merged**